docs/static/releasenotes.asciidoc

```diff
@@ -54,6 +55,98 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-16-0]]
+=== Logstash 7.16.0 Release Notes
+
+[[notable-7-16-0]]
+==== Performance improvements and notable issues fixed
+
+* Add `pipeline.ordered` setting for docker image https://github.com/elastic/logstash/pull/13300[#13300]
+
+**Updates to dependencies**
+
+* Update JRuby to 9.2.20.1 https://github.com/elastic/logstash/pull/13468[#13468]
+
+[[deprecated-7-16-0]]
+==== Deprecations
+
+* Logstash 8.0 will remove support for Java versions before 11, thus using Java 8 has been deprecated.
+
+* The use of the `JAVA_HOME` environment variable is targeted for removal in 8.0, please use `LS_JAVA_HOME` instead.
+
+* Reminder that the Ruby Execution Engine is deprecated and will not be available in Logstash 8.
+  The Java Execution Engine has been the default since 7.0 and reached substantial maturity in 7.10.
+
+[[featured-7-16-0]]
+==== New features and enhancements
+
+* Added options for securing the Logstash HTTP API using TLS and/or HTTP Basic auth https://github.com/elastic/logstash/pull/13308[#13308].
+  We're planning on following up on the feature set shipped in 7.16 for details please see https://github.com/elastic/logstash/issues/13196[#13196].
+
+* Opting in to ECS compatibility per-pipeline (`pipeline.ecs_compatibility`) or globally has graduated from BETA.
+
+* Introduce `LS_JAVA_HOME` environment variable https://github.com/elastic/logstash/pull/13204[#13204].
+  Logstash launch scripts have been using `JAVA_HOME` to identify a JDK configured by the user,
+  as that environment variable could be too generic Logstash now has a dedicated `LS_JAVA_HOME`.
+  If `LS_JAVA_HOME` is not specified Logstash still looks for `JAVA_HOME`.
```

Comment:
```suggestion
* Introduce `LS_JAVA_HOME` environment variable to replace `JAVA_HOME` https://github.com/elastic/logstash/pull/13204[#13204].
 Until 8.0, Logstash looks for `JAVA_HOME` If `LS_JAVA_HOME` is not specified.  (See <<deprecated-7-16-0>>.)
  
  We recommend using the bundled JDK unless you have a compelling reason to use a different one. 
```

Reply:
This was confusing in light of the JAVA_HOME deprecation we listed earlier, so I tried to get the messaging in sync. Feel free to tweak.